javascript/javascript2/week1/spirit-animal.js

```diff
@@ -0,0 +1,38 @@
+const animals = ["The bored panda", "The crying butterfly", " The fullmoon wolf", "The fat beaver", "The bearded kangaroo",
+    "The loonar racoon", "The silke toad", "The bigfooted gazelle", "The seaweed opposum", "The lunar hawk"];
+
+const btn = document.getElementById('button');
+const input = document.getElementById("nameInput"); 
+const radios = document.getElementsByName('type');
+radios.forEach(radio => radio.addEventListener('change', changeType));
```

Comment:
The code doesn't work initially. Either add a click eventlistener from the get-go. Or run the changeType with a type parameter.

Reply:
Added another eventListner